src/formats/ilbm.rs

```diff
@@ -0,0 +1,18 @@
+use crate::{
+    util::{read_u16, Endian},
+    ImageResult, ImageSize,
+};
+use std::io::{BufRead, Seek, SeekFrom};
+
+pub fn size<R: BufRead + Seek>(reader: &mut R) -> ImageResult<ImageSize> {
+    reader.seek(SeekFrom::Start(20))?;
+
+    Ok(ImageSize {
+        width: read_u16(reader, &Endian::Big)? as usize,
+        height: read_u16(reader, &Endian::Big)? as usize,
+    })
+}
+
+pub fn matches(header: &[u8]) -> bool {
+    header.len() >= 12 && &header[0..4] == b"FORM" && &header[8..12] == b"ILBM"
```

Comment:
According to [Wikipedia](https://en.wikipedia.org/wiki/ILBM), it looks like "PBM " is also a valid value for 8..12. Unsure if this changes anything else at the moment.

Reply:
It seems that PBM is just a minor variant without the run-length encoding of the BODY chunk, so won’t affect the parsing of the BMHD chunk. I've added a check for the "PBM " type ID.